doc/iterations/2023.6/shapes/shape Odesign converter.adoc

```diff
@@ -0,0 +1,46 @@
+= (M) ODesign reader
```

Comment:
- Name the file something like `odesign_to_view_converter.adoc`, no need to mention that it's a shape once again and this file may be aggregated in a website so let's not use too many spaces because it creates unreadable URLs
- The title of the shape has nothing to do with an ODesign reader but instead something like `Convert odesign models to view models`